helpful.el

```diff
@@ -1064,16 +1073,18 @@ state of the current symbol."
          (helpful--syntax-highlight source (if primitive-p 'c-mode)))
         ((and (consp source) (eq (car source) 'closure))
          (helpful--syntax-highlight
-          (concat ";; Closure converted to defun by helpful.\n"
+          (concat ";; Closure converted to defun by helpful."
```

Comment:
All these changes are great, and definitely improve the code. My only query is this line: why are you removing the newline here?

Reply:
I guess it was just a casualty of mass `replace-regexp`. I added it back.

Reply:
fixed

Reply:
(I accidentally commented with my work account earlier!)